### DISCUSSION TITLE
Differences with scramble

### DISCUSSION THREAD

**Discussion by darkbasic** [Nova-Edge/laravel-openapi#22] [Q&A] [answered]
Hi,
What are the main differences between this project and https://github.com/dedoc/scramble ?

**Comment by shokme** [comment, marked as answer]
First this project is a fork from an not mainted https://github.com/vyuldashev/laravel-openapi

I do use both for different microservices. I would say this one offer more flexibility but you have to write everything.

Scramble is younger, but can already build good documentation without the needs to do anything (except for example).
This is a big big win of time. The only problem with Scramble is you don't have any control on the schema generated you can't really customise the output.

If you have apis that are not too complex you should use Scramble. If you're not sure use this package.
